packages/dos-donts-block/src/components/AddItemButton.tsx

```diff
@@ -0,0 +1,23 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { joinClassNames, textAndBgColor, textAndBgColorHover } from '@frontify/guideline-blocks-shared';
+import { AddItemButtonProps } from '../types';
+
+const AddItemButton = ({ onClick, label, icon, secondaryLabel }: AddItemButtonProps) => (
```

Comment:
I think you can reuse the same component here (AddImagesButton, or if you rename it BlockInjectButton)

Reply:
I was somehow confused to reuse the other one as there is much more logic in it then in this one. It is just most of the styles it reuses. You know what I mean?

Reply:
You are right. Removed it and made the other block more generic.